XRTK-Core/Packages/com.xrtk.core/Interfaces/Providers/Controllers/IMixedRealityHandController.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+using XRTK.Definitions.Utilities;
+using XRTK.Providers.Controllers.Hands;
+
+namespace XRTK.Interfaces.Providers.Controllers
+{
+    /// <summary>
+    /// Controller definition, used to manage a hand controller.
+    /// </summary>
+    public interface IMixedRealityHandController : IMixedRealityController
+    {
+        /// <summary>
+        /// Gets the collection of bounds available for this hand controller.
+        /// </summary>
+        IReadOnlyDictionary<TrackedHandBounds, Bounds> Bounds { get; }
+
+        /// <summary>
+        /// Gets whether this hand controller is currently tracked.
+        /// </summary>
+        bool IsTracked { get; }
+
+        /// <summary>
+        /// Gets whether the hand is in pointing pose for raycasting.
+        /// </summary>
+        bool IsInPointingPose { get; }
```

Comment:
This doesn't make much sense to me as the only things doing any `Pointing` are the pointers attached to controllers.

I'll have to read up on this a bit more, but it seems misplaced.